### PR TITLE
docs: support SkipDecoder

### PR DESCRIPTION
**Pull request by DMwangnima**
#### What type of PR is this?
docs

#### Check the PR title.

- [ ] This PR title match the format: \<type\>(optional scope): \<description\>
- [ ] The description of this PR title is user-oriented and clear enough for others to understand.

#### (Optional) Translate the PR title into Chinese.
docs: 支持 SkipDecoder

#### (Optional) More detailed description for this PR(en: English/zh: Chinese).

en:
SkipDecoder using docs.
zh(optional): 
SkipDecoder 使用文档

### CODE REVIEW

File: content/zh/docs/kitex/Tutorials/code-gen/skip_decoder.md [hunk 1]
```diff
@@ -0,0 +1,158 @@
+---
+title: "SkipDecoder"
+date: 2024-06-20
+weight: 8
+keywords: ["SkipDecoder", "精简产物"]
+description: "开启SkipDecoder，突破Frugal与FastCodec的协议限制，精简代码产物"
```
Comment (GuangmingLuo) [inline content/zh/docs/kitex/Tutorials/code-gen/skip_decoder.md:6]: 中英文之间加一下空格

File: content/zh/docs/kitex/Tutorials/code-gen/skip_decoder.md [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+---
+
+# 背景
+
+对于 Thrift 序列化协议，当前为了更好的性能基本都搭配 Thrift Framed 传输协议(e.g. TTHeader, TTHeaderFramed) 使用 FastCodec 或者 Frugal。但为了兼容少量使用 Thrift Buffered 传输协议的服务，Kitex 仍然会生成符合官方接口要求的 Apache Thrift Codec 代码。(Framed 与 Buffered 协议详情请参考 https://github.com/apache/thrift/blob/master/doc/specs/thrift-rpc.md#framed-vs-unframed-transport)
```
Comment (GuangmingLuo) [inline content/zh/docs/kitex/Tutorials/code-gen/skip_decoder.md:11]: 换成超链接形式：详情请参考 \[Framed 与 Buffered 协议\](https://github.com/apache/thrift/blob/master/doc/specs/thrift-rpc.md#framed-vs-unframed-transport)

File: content/zh/docs/kitex/Tutorials/code-gen/skip_decoder.md [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+    import (
+        "github.com/cloudwego/kitex/pkg/remote/codec/thrift"
+        "demo/kitex_gen/kitex/samples/echo/echoservice"
+    )
+    
+    func main() {
+        cli := echoservice.MustNewClient("kitex.samples.echo", 
+            client.WithPayloadCodec(thrift.NewThriftCodecWithConfig(thrift.FastRead|thrift.FastWrite|thrift.EnableSkipDecoder)),
+        )
+    }
```
Comment (GuangmingLuo) [inline content/zh/docs/kitex/Tutorials/code-gen/skip_decoder.md:71]: 这里没有使用 code block

Comment (GuangmingLuo) [reply content/zh/docs/kitex/Tutorials/code-gen/skip_decoder.md:71]: 下同

File: content/zh/docs/kitex/Tutorials/code-gen/skip_decoder.md [hunk 4]
```diff
@@ -0,0 +1,158 @@
+---
+title: "SkipDecoder"
+date: 2024-06-20
+weight: 8
+keywords: ["SkipDecoder", "精简产物"]
+description: "开启SkipDecoder，突破Frugal与FastCodec的协议限制，精简代码产物"
+---
+
+# 背景
```
Comment (GuangmingLuo) [inline content/zh/docs/kitex/Tutorials/code-gen/skip_decoder.md:9]: 文章主体里面最高用二级标题，##

**Review by GuangmingLuo — DISMISSED** [review]

File: content/en/docs/kitex/Tutorials/code-gen/skip_decoder.md [hunk 5]
```diff
@@ -0,0 +1,175 @@
...
+title: "SkipDecoder"
+date: 2024-06-20
+weight: 10
+keywords: ["SkipDecoder", "Simplify Generated Code"]
+description: "Enable SkipDecoder to break the protocol limitations of Frugal and FastCodec, and simplify the generated code"
+---
+
+## Background
+
+For Thrift serialisation protocols, the current Thrift Framed transport (e.g. TTHeader, TTHeaderFramed) uses FastCodec or Frugal for better performance, but for the sake of compatibility with the small number of services that use the Thrift Buffered transport, Kitex Kitex still generates Apache Thrift Codec code that conforms to the official interface requirements. (See [Framed and Buffered protocols](https://github.com/apache/thrift/blob/master/doc/specs/thrift-rpc.md#framed-vs-unframed-transport) for details).
```
Comment (rogerogers) [inline content/en/docs/kitex/Tutorials/code-gen/skip_decoder.md:11]: 这里有两个 Kitex

Comment (DMwangnima) [reply content/en/docs/kitex/Tutorials/code-gen/skip_decoder.md]: done.

**Review by ppzqh — APPROVED** [review]

**Pull request merged**